src/Workspaces/Core/Portable/Options/IGlobalOptionService.cs

```diff
@@ -43,21 +43,19 @@ internal interface IGlobalOptionService : IOptionsReader
         /// Sets and persists the value of a global option.
         /// Sets the value of a global option.
         /// Invokes registered option persisters.
-        /// Triggers <see cref="OptionChanged"/>.
+        /// Triggers option changed event for handlers registered with <see cref="AddOptionChangedHandler"/>.
         /// </summary>
         void SetGlobalOption(OptionKey2 optionKey, object? value);
 
         /// <summary>
         /// Atomically sets the values of specified global options. The option values are persisted.
-        /// Triggers <see cref="OptionChanged"/>.
+        /// Triggers option changed event for handlers registered with <see cref="AddOptionChangedHandler"/>.
         /// </summary>
         /// <remarks>
         /// Returns true if any option changed its value stored in the global options.
         /// </remarks>
         bool SetGlobalOptions(ImmutableArray<KeyValuePair<OptionKey2, object?>> options);
 
-        event EventHandler<OptionChangedEventArgs>? OptionChanged;
```

Comment:
unclear if this will impact partners. need to probably check on TS at least since they aren't EA on everything.